workarea-product_badges.gemspec

```diff
@@ -15,5 +15,5 @@ Gem::Specification.new do |s|
 
   s.license = 'Business Software License'
 
-  s.add_dependency "workarea", "~> 3.x", ">= 3.4.x"
+  s.add_dependency "workarea", "~> 3.x"
```

Comment:
So by changing this, we're saying it's compatible with all v3 versions - do we know that's true?